src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp

```diff
@@ -22,46 +22,34 @@ namespace NumericalFluxes {
  * \ingroup NumericalFluxesGroup
  * \brief Compute the HLL numerical flux.
  *
- * Let \f$U\f$ and \f$F^i(U)\f$ be the state vector of the system and its
- * corresponding volume flux, respectively. Let \f$n_i\f$ be the unit normal to
- * the interface. Defining \f$F_n(U) := n_i F^i(U)\f$, and denoting the
- * corresponding projections of the numerical fluxes as \f${F_n}^*(U)\f$, the
- * HLL flux \ref hll_ref "[1]" for each variable is
+ * Let \f$U\f$ be the state vector of the system and \f$F^i\f$ the corresponding
+ * volume fluxes. Let \f$n_i\f$ be the unit normal to
+ * the interface. Denoting \f$F := n_i F^i\f$, the  HLL flux is \cite Harten1983
  *
  * \f{align*}
- * {F_n}^*(U) = \frac{c_\text{max} F_n(U_\text{int}) -
- * c_\text{min} F_n(U_\text{ext})}{c_\text{max} - c_\text{min}}
- * - \frac{c_\text{min}c_\text{max}}{c_\text{max} - c_\text{min}}
+ * G_\text{HLL} = \frac{S_\text{max} F_\text{int} -
+ * S_\text{min} F_\text{ext}}{S_\text{max} - S_\text{min}}
+ * - \frac{S_\text{min}S_\text{max}}{S_\text{max} - S_\text{min}}
  * \left(U_\text{int} - U_\text{ext}\right)
  * \f}
  *
  * where "int" and "ext" stand for interior and exterior, respectively.
- * \f$c_\text{min}\f$ and \f$c_\text{max}\f$ are estimates on the minimum
- * and maximum signal velocities bounding the interior-moving and
- * exterior-moving wavespeeds that arise when solving the Riemann problem.
- * Here we use the simple estimates \ref estimates_ref "[2]"
+ * \f$S_\text{min}\f$ and \f$S_\text{max}\f$ are estimates on the minimum
+ * and maximum signal velocities bounding the ingoing and outgoing wavespeeds
+ * that arise when solving the Riemann problem. Here we use the simple
+ * estimates \cite Davis1988
  *
- * \f{align*}
- * c_\text{min} &= \text{min}\left( \lambda_1(U_\text{int}; n_\text{int}),
- * \lambda_1(U_\text{ext}; n_\text{ext}), 0\right),\\
- * c_\text{max} &= \text{max}\left( \lambda_N(U_\text{int}; n_\text{int}),
- * \lambda_N(U_\text{ext}; n_\text{ext}), 0\right),
+ *\f{align*}
```

Comment:
Ping on reinserting this whitespace. Squash.

Reply:
Done.